scripts/router/modules/46-confirm-ui.ps1

```
@@ -371,6 +376,22 @@ function Build-ConfirmSkillOptions {
+    if ($selectedSkill -and -not (@($ranking | ForEach-Object { [string]$_.skill }) -contains $selectedSkill)) {
+        $selectedRow = $null
+        if ($packRow -and $packRow.stage_assistant_candidates) {
+            $selectedRow = @($packRow.stage_assistant_candidates | Where-Object { [string]$_.skill -eq $selectedSkill } | Select-Object -First 1)
+        }
+        if (-not $selectedRow -and $packRow -and $packRow.candidate_ranking) {
+            $selectedRow = @($packRow.candidate_ranking | Where-Object { [string]$_.skill -eq $selectedSkill } | Select-Object -First 1)
+        }
+        if (-not $selectedRow) {
+            $selectedRow = [pscustomobject]@{
+                skill = $selectedSkill
+                score = if ($Result.selected.selection_score -ne $null) { [double]$Result.selected.selection_score } else { 0.0 }
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)